Routes.js

```diff
@@ -0,0 +1,41 @@
+const express = require("express")
+const app = express()
+const port =  3001
+
+const villanInfo = [
+    {   
+        id:1,
+        seriesMovieName:"Looney Tune",
+        villainName:"Wile E. Coyote",
+        actions:"This cunning but perpetually unlucky coyote spends all his cartoon life trying to capture the Road Runner, only to be foiled by his own outlandish contraptions and hilarious mishaps. From falling anvils to exploding rockets, Wile E.'s plans are masterpieces of ineptitude.",
+        villainImgLink:"https://static.wikia.nocookie.net/looneytunes/images/7/7f/Coyote.gif/revision/latest/thumbnail/width/360/height/360?cb=20060219181853",
+        posterLink:"https://m.media-amazon.com/images/M/MV5BMDEwNTAwNjMtNTgwZC00Y2RjLTliOTYtZWI4NGNlNjI4ZTVjXkEyXkFqcGdeQXVyMTA0MTM5NjI2._V1_.jpg"
+    }
+]
+app.use(express.json());
+app.get('/', (req, res) => {
+    res.send(villanInfo)
+})
+
+app.post('/', (req, res) => {
+    const newVillain= req.body;
+    villanInfo.push(newVillain)
+    res.send(villanInfo)
+
+})
+app.put('/:id', (req, res) => {
```

Comment:
The id in the DELETE and PUT requests is parsed from the request parameters, but there is no check to ensure that this id is valid. Consider adding a check to ensure that the id is valid before proceeding with the operation. [important]